main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_info["race"]["name"],
+            description=player_info["race"]["description"]
+        )
+
+        for skill in player_info["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+
+        guild, _ = Guild.objects.get_or_create(
+            name=player_info["guild"]["name"],
+            description=player_info["guild"]["description"]
+        ) if player_info["guild"] else (None, False)
```

Comment:
The use of a conditional expression to handle the possibility of a missing guild is a good approach. However, ensure that the JSON structure always includes a 'guild' key, even if it's null, to avoid potential KeyError exceptions.